### DISCUSSION TITLE
如何定义每个节点对应不同端口？

### DISCUSSION THREAD

**Discussion by dnlzsy** [v2rayA/v2rayA#1445] [Q&A]
我又两个节点分别是a和b，我想a运行在2222端口，b运行在3333端口，这样，我客户端就可以根据不同用途自己选择是通过a出去还是通过b出去。

**Comment by mxdabc** [comment]
我也有这样的疑问，目前是手动运行xray

**Comment by eronez** [comment]
Just fork and develop your flavor.

**Reply by mxdabc** [reply]
Can you provide some ideas? For example, the process is multi-open？

**Comment by eronez** [comment]
https://v2raya.org/docs/advanced-application/multiple-outbound-routing
@dnlzsy try this

**Reply by mxdabc** [reply]
Thanks.
